src/GeositeFramework/js/Plugin.js

```diff
@@ -401,6 +405,10 @@ require(['use!Geosite',
             view.$uiContainer = $uiContainer;
 
             $uiContainer
+                // Call `hibernate` method on plugin when the eye icon is clicked
+                .find('.plugin-eye').on('click', function () {
+                    model.hibernate();
+                }).end()
```

Comment:
I would check to make sure this isn't already satisfied by https://github.com/CoastalResilienceNetwork/GeositeFramework/blob/c3d2e103da69463456dbaf738bb0e86ec86fc657/src/GeositeFramework/js/Plugin.js#L417.